zookeeper-client/zookeeper-client-c/src/cli.c

```diff
@@ -840,10 +877,26 @@ int main(int argc, char **argv) {
                 "-s, --ssl <ssl params>         Comma separated parameters to initiate SSL connection\n"
                 "                                 e.g.: server_cert.crt,client_cert.crt,client_priv_key.pem,passwd\n"
 #endif
+#ifdef HAVE_CYRUS_SASL_H
+                "-z, --service <service>        SASL service parameter (default: 'zookeeper')\n"
+                "-o, --server-fqdn <fqdn>       SASL host name (default: reverse DNS lookup)\n"
+                "-n, --mechlist <mechlist>      Comma separated list of SASL mechanisms\n"
+                "-u, --user <user>              SASL user name\n"
+                "-l, --realm <realm>            SASL realm\n"
+                "-p, --password-file <file>     File containing the password for SASL\n"
+#endif /* HAVE_CYRUS_SASL_H */
                 "-r, --readonly                 Connect in read-only mode\n"
                 "-d, --debug                    Activate debug logs right from the beginning (you can also use the \n"
                 "                                 command 'verbose' later to activate debug logs in the cli shell)\n\n",
                 argv[0]);
+#ifdef HAVE_CYRUS_SASL_H
+        fprintf(stderr,
+                "SASL EXAMPLES:\n"
```

Comment:
please also mention here, that SSL and SASL are not working together at the moment. AFAICS in the implementation, if the user defines both SSL and SASL parameters, then only the SASL will be used. (I am not sure what happens, if the SASL fails to authenticate... with in this case fallback to SSL?)

Reply:
Done.  (But see the note above above, about `zookeeper_init_with_options`.)